nncf/torch/graph/transformations/command_creation.py

```diff
@@ -0,0 +1,31 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.torch.graph.transformations.commands import PTBiasCorrectionCommand
+from nncf.torch.graph.transformations.commands import PTTargetPoint
+
+
+def create_bias_correction_command(node: NNCFNode, bias_value: np.ndarray) -> PTBiasCorrectionCommand:
```

Comment:
The `PTBiasCorrectionCommand` instance supports only `bias_value` as `torch.Tensor`, Here is the `np.ndarray`.